config/initializers/active_admin.rb

```diff
@@ -6,18 +6,6 @@
   #
   config.site_title = 'David Runger Admin'
 
-  # Set the link url for the title. For example, to take
-  # users to your main site. Defaults to no link.
-  #
-  # config.site_title_link = "/"
```

Comment:
The various configs being deleted here are all mentioned as having been removed at https://github.com/activeadmin/activeadmin/blob/master/UPGRADING.md .